patterns/common/sailfish-device-capabilities.yaml.template

```diff
@@ -0,0 +1,8 @@
+Description: All available capabilities. (Template file lists all, final .yaml will reflect only what device needs)
+Name: sailfish-device-capabilities
+Requires:
+%have_vibrator- pattern:sailfish-ha-vibrator
```

Comment:
I would just drop this file completely and add these two to jolla-configuration-@DEVICE@.yaml or so.